sys/include/net/lwm2m.h

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (C) 2018 Beduino Master Projekt - University of Bremen
+ *               2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup     net_lwm2m LwM2M
+ * @ingroup      net
+ * @brief        Lightweight Machine to Machine (LwM2M) definitions
+ * @{
+ *
+ * @file
+ * @brief       LwM2M header definitions
+ *
+ * @author      Christian Manal <manal@uni-bremen.de>
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+
+
+#ifndef NET_LWM2M_H
+#define NET_LWM2M_H
```

Comment:
Is it important that this file is in `sys/include/net`? Why not in the pkg `include`? Should we ever decide to implement our own LwM2M, this file name likely would be a conflict here.

Reply:
It's not particularly important. Although, these parameters are not specific to the Wakaama implementation. In the case of a different implementation we would also need to configure them somehow.

I would be ok moving the header to the package. In that case, should the prefix be `WAKAAMA_` instead?

Reply:
Another option might be to rename it to something less generic -- `wakaama.h`?

Reply:
Too funny -- that's the second time I didn't see your comment. If you move the file to the pkg, I don't think it's necessary to change the prefix. I can't imagine an application actually using the package and a RIOT native implementation at the same time. My main concern was the name of the file in this location.

Reply:
I moved the header to `pkg/wakaama/include/lwm2m_client_config.h`

Reply:
Great choice for the name of the file!